### DISCUSSION TITLE
Tablist Spectator mode *(No move option)*

### DISCUSSION THREAD

**Discussion by dmnchild** [Zrips/CMI#1902] [Feature Requests]
Looking for option to not move people in spectator mode. Makes it obvious when staff is snooping/watching people for xray and such. I know TAB has an option so it is possible, jsut not sure how much work involved. Currently it uses the default option and moves spectators to bottom of list. 

---

**Comment by CloudeLecaw** [comment]
what about using cmis vanish instead of spectator(or combined with it)
and it also has a permissions to see vanish people in tab and infront of you.
